string	                          pose_name
geometry_msgs/PoseWithCovariance  pose
---
bool	                            error
string          	                description
